coordinator/config.go

```diff
@@ -50,7 +50,16 @@ func ReadConfig(configFilePath string) (*Config, error) {
 	}
 
 	if err := v.ReadInConfig(); err != nil {
-		return nil, errors.Wrap(err, "unable to read configuration file")
+		// If we can't find the config let's rely on the default one.
```

Comment:
Should we just refactor this to a helper function ? The same code kinda gets repeated everywhere. We probably just need to pass the viper instance and the config name.

Reply:
I've considered this but as soon as I thought about having it in `config` alongside the JSON files I changed my mind.